lib/particular-calendars/canada.ts

```diff
@@ -1,88 +1,119 @@
 import { PatronTitles } from '../constants/martyrology-metadata';
 import { Precedences } from '../constants/precedences';
 import { CalendarDef } from '../models/calendar-def';
-import { Inputs } from '../types/calendar-def';
+import { Inputs, ParticularConfig } from '../types/calendar-def';
 import { Americas } from './americas';
 
 export class Canada extends CalendarDef {
   parentCalendar = Americas;
 
+  particularConfig: ParticularConfig = {
+    epiphanyOnSunday: true,
+    ascensionOnSunday: true,
```

Comment:
We could use [this]([this](https://www.cccb.ca/wp-content/uploads/2023/09/2023-2024.pdf#page=6)) as the source for this.

Reply:
I guess you forgot the link to the source here 😃

Reply:
No, I didn’t, I’ve just messed with the Markdown format (actually, I didn’t notice GitHub added the link in MD format :wink:). I have just fixed it. :wink: